salsify_uuid.go

```diff
@@ -0,0 +1,38 @@
+package goavro
+
+import (
+	"encoding/hex"
+	"fmt"
+)
+
+func salsifyTextualFromNative(buf []byte, d interface{}) ([]byte, error) {
```

Comment:
Should we specify this is a binary salsify UUID in these function names?

```suggestion
func salsifyUuidBinaryTextualFromNative(buf []byte, d interface{}) ([]byte, error) {
```

Reply:
I was a bit hesitant, given other methods in the interface reference the term 'binary'. I'll give it some thought